src/main/java/org/minions/devfund/cristhian_arispe/CountCharacters.java

```diff
@@ -0,0 +1,32 @@
+package org.minions.devfund.cristhian_arispe;
+
+/**
+ * This is the solution for the first kata.
+ */
+public final class CountCharacters {
+
+    /**
+     * CountCharacters constructor.
+     */
+    private CountCharacters() {
+    }
+
+    /**
+     * Count characters on the String.
+     * @param str String value.
+     * @param c Char value.
+     * @return The number of characters that are found in the string.
+     */
+    public static int charCount(final String str, final char c) {
+        String[] wordArray = str.split("");
+        int counter = 0;
+        for (int i = 0; i < str.length(); i++) {
+            if (wordArray[i].toLowerCase().equals((c + "").toLowerCase())) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)